.github/workflows/lib-tests.yaml

```diff
@@ -31,6 +31,7 @@ jobs:
   tests:
     runs-on: ubuntu-latest
     needs: changes
+    if: needs.changes.outputs.libraries != '[]' && needs.changes.outputs.libraries != ''
```

Comment:
Why is this check necessary?

Reply:
If there weren't any changes, the libs workflow was being marked as failed  (see e.g. [here](https://github.com/ML4GW/aframe/actions/runs/16118179246))